dom/events.ts

```diff
@@ -1,22 +1,27 @@
 import {splitStringValue} from "./utils";
-const listenerRegistry = new window.WeakMap();
+const listenerRegistry = new WeakMap();
+
+/**
+ * Custom event listener token, that can unregister delegated or once event listeners
+ */
+export type EventIntermediateToken = (any : any) => void;
+
+/**
+ * Event handler for delegated events
+ */
+export type DelegatedEventHandler = (Event : Event, HTMLElement : HTMLElement) => void;
```

Comment:
Can we add JS Doc here?

So that we can describe, that the first is the original event and the second is the matched element, where the delegation was filtered to.